### PR TITLE
#27500 Prepare Integration module Tests for PHPUnit 8

### PR DESCRIPTION
**Pull request by lbajsarowicz**
### Description (*)
Preparing PHPUnit Tests to run with PHPUnit 8.
PRs for each module is separated to make tests easier to merge.

### Fixed Issues (if relevant)
1. magento/magento2#27500

### Manual testing scenarios (*)
N/A

### Questions or comments
PR #27701 needs to be merged first (otherwise PHPUnit tests fail)

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Comment by ihor-sviziev** [comment]
@magento run Functional Tests B2B

File: app/code/Magento/Integration/Test/Unit/Controller/Adminhtml/IntegrationTest.php [hunk 1]
```diff
@@ -335,23 +361,22 @@ protected function _createIntegrationController($actionName)
     protected function _verifyLoadAndRenderLayout()
     {
         $map = [
-            [\Magento\Framework\App\Config\ScopeConfigInterface::class, $this->_configMock],
-            [\Magento\Backend\Model\Layout\Filter\Acl::class, $this->_layoutFilterMock],
```
Comment (ihor-sviziev) [inline app/code/Magento/Integration/Test/Unit/Controller/Adminhtml/IntegrationTest.php:339]: Why it was removed?

File: app/code/Magento/Integration/Test/Unit/Model/IntegrationServiceTest.php [hunk 2]
```diff
@@ -67,7 +67,7 @@ protected function setUp()
             Integration::ID => self::VALUE_INTEGRATION_ID,
             Integration::NAME => self::VALUE_INTEGRATION_NAME,
             Integration::EMAIL => self::VALUE_INTEGRATION_EMAIL,
-            Integration::EMAIL => self::VALUE_INTEGRATION_ENDPOINT,
```
Comment (ihor-sviziev) [inline app/code/Magento/Integration/Test/Unit/Model/IntegrationServiceTest.php:70]: Good catch!

**Review by ihor-sviziev — CHANGES_REQUESTED** [review]

File: app/code/Magento/Integration/Test/Unit/Controller/Adminhtml/IntegrationTest.php (hunk 1 above)
Comment (ihor-sviziev) [reply app/code/Magento/Integration/Test/Unit/Controller/Adminhtml/IntegrationTest.php:339]: Ah, I see it wasn't used

**Review by ihor-sviziev — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @ihor-sviziev, thank you for the review.
[ENGCOM-7444](https://jira.corp.magento.com/browse/ENGCOM-7444) has been created to process this Pull Request

**Pull request merged**